cmd/temporal-lens/main.go

```diff
@@ -68,7 +68,15 @@ var commands = map[string]cmd.Cmd{
 
 			// create lens v2 service
 			l.Info("instantiating Lens V2")
-			srv, err := lens.NewV2(lens.V2Options{}, manager, tf, l)
+			srv, err := lens.NewV2(lens.V2Options{
+				Engine: engine.Opts{
+					StorePath: cfg.Lens.Options.Engine.StorePath,
+					Queue: queue.Options{
+						Rate:      time.Duration(cfg.Lens.Options.Engine.Queue.Rate) * time.Second,
```

Comment:
hm can't this just be:

```suggestion
						Rate:      cfg.Lens.Options.Engine.Queue.Rate * time.Second,
```

since I'm pretty sure `3 * time.Second` is valid 🤔

Reply:
[nope](https://stackoverflow.com/questions/17573190/how-to-multiply-duration-by-integer) :( tbh the `time` api is pretty frustrating to work with lmao

Reply:
ah i see lame

Reply:
oh well, nothing we can do here

Reply:
it works here though? https://sourcegraph.com/github.com/RTradeLtd/Lens/-/blob/engine/engine_test.go#L58:40